ghost/stack.yml

```
@@ -5,22 +5,32 @@ services:
   ghost:
     image: ghost:4-alpine
     restart: always
+    depends_on:
+      - ghost-db
     ports:
       - 8080:2368
     environment:
       # see https://ghost.org/docs/config/#configuration-options
       database__client: mysql
-      database__connection__host: db
+      database__connection__host: ghost-db
       database__connection__user: root
       database__connection__password: example
       database__connection__database: ghost
       # this url value is just an example, and is likely wrong for your environment!
       url: http://localhost:8080
       # contrary to the default mentioned in the linked documentation, this image defaults to NODE_ENV=production (so development mode needs to be explicitly specified if desired)
       #NODE_ENV: development
+    volumes:
+      - some-ghost-data:/var/lib/ghost/content
 
-  db:
+  ghost-db:
     image: mysql:8.0
     restart: always
     environment:
       MYSQL_ROOT_PASSWORD: example
+    volumes:
+      - some-ghost-mysql:/var/lib/mysql
+
+volumes:
+  some-ghost-data:
+  some-ghost-mysql:
```

We don't have any other `stack.yml` files that use this naming scheme (we use it in the prose so it's more obviously an example that should be modified, where here the entire file is that).

```suggestion
      - db:/var/lib/mysql

volumes:
  ghost:
  db:
```